lightningd/closing_control.c

```diff
@@ -747,11 +790,28 @@ static struct command_result *json_close(struct command *cmd,
 					msg = towire_dualopend_send_shutdown(
 						NULL,
 						channel->shutdown_scriptpubkey[LOCAL]);
-				} else
+				} else {
+					struct ext_key ext_key_val;
+					struct ext_key *final_ext_key = NULL;
+					if (final_index) {
```

Comment:
this seems to be always true

Reply:
`closing_control.c:669` "user specified late addr which is not in the wallet"